Sources/StreamChat/ChatClient.swift

```diff
@@ -480,6 +581,15 @@ extension ClientError {
             """
         }
     }
+    
+    public class ConnectionWasNotInitiated: ClientError {
+        override public var localizedDescription: String {
+            """
+                Before performing any other actions on chat client it's required to connect by using
```

Comment:
nit: You can use `\` at the end of the line to prevent an incorrect new line when printing the message.